pkg/cluster/provider.go

```diff
@@ -169,7 +169,7 @@ func ProviderWithPodman() ProviderOption {
 }
 
 // Create provisions and starts a kubernetes-in-docker cluster
-// TODO: move name to an option to override config
+// TODO: Replace by CreateWithOpts in the next major version
 func (p *Provider) Create(name string, options ...CreateOption) error {
 	// apply options
 	opts := &internalcreate.ClusterOptions{
```

Comment:
Just commenting this as an option to think about if you need to do any further updates - it might be nice to now make `Create` just be a light wrapper that takes the separate `name` argument and use `CreateWithName` to just call out to `CreateWithOpts` and force everything through the one common code path.

Not a big deal at all though, so feel free to ignore this. ;)

Reply:
ok. Thanks very much.